src/controllers/issuer.ts

```diff
@@ -544,41 +605,131 @@ export class IssuerController {
+	 *         description: Fetch a DID-Linked Resource by Resource ID unique identifier. Since this is a unique identifier, other Resource query parameters are not required. See <a href="https://docs.cheqd.io/identity/credential-service/did-linked-resources/understanding-dlrs/technical-composition">DID-Linked Resources</a> for more details.
+	 *         schema:
+	 *           type: string
+	 *           format: uuid
+	 *         example: 3ccde6ba-6ba5-56f2-9f4f-8825561a9860
+	 *       - in: query
+	 *         name: resourceName
+	 *         description: Filter a DID-Linked Resource query by Resource Name. See <a href="https://docs.cheqd.io/identity/credential-service/did-linked-resources/understanding-dlrs/technical-composition">DID-Linked Resources</a> for more details.
+	 *         schema:
+	 *           type: string
+	 *         example: Cheqd Issuer Logo
+	 *       - in: query
+	 *         name: resourceType
+	 *         description: Filter a DID-Linked Resource query by Resource Type. See <a href="https://docs.cheqd.io/identity/credential-service/did-linked-resources/understanding-dlrs/technical-composition">DID-Linked Resources</a> for more details.
+	 *         schema:
+	 *           type: string
+	 *         example: IssuerLogo
+	 *       - in: query
+	 *         name: resourceVersion
+	 *         description: Filter a DID-Linked Resource query by Resource Version, which is an optional free-text field used by issuers (e.g., "v1", "Final Version", "1st January 1970" etc). See <a href="https://docs.cheqd.io/identity/credential-service/did-linked-resources/understanding-dlrs/technical-composition">DID-Linked Resources</a> for more details.
+	 *         schema:
+	 *           type: string
+	 *       - in: query
+	 *         name: resourceVersionTime
+	 *         description: Filter a DID-Linked Resource query which returns the closest version of the Resource *at* or *before* specified time. See <a href="https://docs.cheqd.io/identity/credential-service/did-linked-resources/understanding-dlrs/technical-composition">DID-Linked Resources</a> for more details.
+	 *         schema:
+	 *           type: string
+	 *           format: date-time
+	 *         example: 1970-01-01T00:00:00Z
+	 *       - in: query
+	 *         name: resourceMetadata
+	 *         description: Return only metadata of DID-Linked Resource instead of actual DID-Linked Resource. Mutually exclusive with some of the other parameters.
+	 *         schema:
+	 *           type: boolean
+	 *       - in: query
+	 *         name: checksum
+	 *         description: Request integrity check against a given DID-Linked Resource by providing a SHA-256 checksum hash. See <a href="https://docs.cheqd.io/identity/credential-service/did-linked-resources/understanding-dlrs/technical-composition">DID-Linked Resources</a> for more details.
+	 *         schema:
+	 *           type: string
+	 *         example: dc64474d062ed750a66bad58cb609928de55ed0d81defd231a4a4bf97358e9ed
```

Comment:
**This** is the section I wanted to be moved to `/resource/search` when I described it to @anikitinDSR. These parameters are all related just to resource-related queries. The ones that remain are all DID-specific. :)

Reply:
Done! ✅